tools/codegen/schema/schema_file_test.go

```diff
@@ -122,9 +125,17 @@ func TestSchemaGenerationFromCodeSpec(t *testing.T) {
 							ComputedOptionalRequired: codespec.ComputedOptional,
 							SetNested: &codespec.SetNestedAttribute{
 								NestedObject: codespec.NestedAttributeObject{
-									Attributes: []codespec.Attribute{
-										stringAttr,
-									},
+									Attributes: []codespec.Attribute{stringAttr, intAttr},
+								},
+							},
+						},
+						{
+							Name:                     "map_nested_attribute",
+							Description:              admin.PtrString("map nested attribute"),
```

Comment:
Why not `conversion.StringPtr`? Instead of using admin?

Reply:
don't really see the need of having conversion.StringPtr in our repo if its available in the SDK, but I do see usage is very mixed 256 results (admin.PtrString) vs 362 results (conversion.StringPtr). Any strong opinion on aligning for one across our repo? cc: @lantoli